blivet/devices/lvm.py

```diff
@@ -2048,7 +2042,7 @@ def _setup(self, orig=False):
         """ Open, or set up, a device. """
         log_method_call(self, self.name, orig=orig, status=self.status,
                         controllable=self.controllable)
-        blockdev.lvm.lvactivate(self.vg.name, self._name)
+        blockdev.lvm.lvactivate(self.vg.name, self._name, ignore_skip=True)
```

Comment:
You can drop the `@type_specific` decorator if this becomes the only version of the `_setup()` method.